install.sh

```diff
@@ -169,9 +169,9 @@ BOOTSTRAP_PACKAGES='zstd crew_mvdir ruby git ca_certificates libyaml openssl'
 [[ "${ARCH}" == "i686" ]] && BOOTSTRAP_PACKAGES+=' zlibpkg gcc_lib'
 
 if [[ -n "${CHROMEOS_RELEASE_CHROME_MILESTONE}" ]]; then
-  sudo cp /lib$LIB_SUFFIX/libc.so* /usr/local/
-  sudo chown chronos /usr/local/lib$LIB_SUFFIX/libc.so*
-  sudo chmod 777 /usr/local/lib$LIB_SUFFIX/libc.so*
+  sudo cp /lib$LIB_SUFFIX/libc.so* $CREW_PREFIX/lib$LIB_SUFFIX
+  sudo chown chronos $CREW_PREFIX/lib$LIB_SUFFIX/libc.so*
+  sudo chmod 777 $CREW_PREFIX/lib$LIB_SUFFIX/libc.so*
```

Comment:
Can we change the 777 to 644?  These libraries should not need execute or other group write permission.

Reply:
oh

Reply:
What are your thoughts about this proposed change @satmandu and @supechicken?

Reply:
so that we will have to make less every time google makes 100 chromeos updates